### PR TITLE
all done

### PR DESCRIPTION
**Pull request by DemiGGV**
06 HW finished. Please check.

### CODE REVIEW

File: helpers/sendMail.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// const { HttpError } = require("../helpers");
+const { SERVER_URL, UN_PASS } = process.env;
+
+const sendMail = async (email, verificationToken) => {
+  const config = {
+    host: "smtp.ukr.net",
+    port: 465,
+    secure: true,
+    auth: {
+      user: "goit_nodejs77@ukr.net",
```
Comment (ceREAL133) [inline helpers/sendMail.js]: this should be in .env file. Email is confident info

File: helpers/sendMail.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      pass: UN_PASS,
+    },
+  };
+
+  const transporter = nodemailer.createTransport(config);
+  const emailOptions = {
+    from: "goit_nodejs77@ukr.net",
```
Comment (ceREAL133) [inline helpers/sendMail.js]: same here

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: helpers/sendMail.js (hunk 1 above)
Comment (DemiGGV) [reply helpers/sendMail.js]: Fixed

File: helpers/sendMail.js (hunk 2 above)
Comment (DemiGGV) [reply helpers/sendMail.js]: Fixed

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**